src/rgw/rgw_main.cc

```diff
@@ -1276,6 +1282,7 @@ int main(int argc, const char **argv)
   curl_global_cleanup();
 
   rgw_perf_stop(g_ceph_context);
+  percentile_perf_stop(g_ceph_context);
```

Comment:
why is this function called separately and not called from within rgw_perf_stop itself?